packages/react/src/drafts/ActionBar/ActionBar.test.tsx

```diff
@@ -33,3 +33,11 @@ describe('ActionBar', () => {
     expect(results).toHaveNoViolations()
   })
 })
+
+/* Test suite
+1. Did it render all the iconbuttons?
```

Comment:
I remember from UnderlineNav and it was easier to write those visual tests in Playwright than Jest but it is up to you 😊

Reply:
Yes, I think all the responsive stuff is better off tested in playwright. It's definitely harder(is it even possible?) to wrangle it in jest.
Will make a better division of test cases!